### ISSUE TITLE
gettext-extractor fails when glob matches a directory

### ISSUE THREAD

**Issue by federicobond** [connectedcars/gettext-extractor#63]
I had a directory named `chart.js` which caused gettext-extractor to try to incorrectly read it as a file. I had to console.log every matched fileName to find the culprit. Do you think it may make sense to add special handling for this case?

```
node:fs:756
  handleErrorFromBinding(ctx);
  ^

Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (node:fs:756:3)
    at tryReadSync (node:fs:437:20)
    at Object.readFileSync (node:fs:483:19)
    at JsParser.parseFile (/Users/federicobond/code/signatura-connect/node_modules/gettext-extractor/dist/parser.js:58:29)
    at JsParser.parseFilesGlob (/Users/federicobond/code/signatura-connect/node_modules/gettext-extractor/dist/parser.js:66:18)
    at file:///Users/federicobond/code/signatura-connect/js-gettext.mjs:21:4
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25) {
  errno: -21,
  syscall: 'read',
  code: 'EISDIR'
}
```

**Comment by lukasgeiter** [comment]
Fixed in version 3.7.2

**Comment by federicobond** [comment]
Thank you!
